src/xma/src/xmaapi/xma_utils.cpp

```diff
@@ -550,46 +611,51 @@ int32_t check_all_execbo(XmaSession s_handle) {
             for (i = 0; i < num_execbo; i++) {
                 auto& ebo = priv1->kernel_execbos[i];
                 if (ebo.in_use) {
-                    ert_start_kernel_cmd *cu_cmd = 
-                        (ert_start_kernel_cmd*)ebo.data;
-                    if (cu_cmd->state == ERT_CMD_STATE_COMPLETED) {
+                    ert_start_kernel_cmd * cu_cmd_pkt =
+                        reinterpret_cast<ert_start_kernel_cmd*>(ebo.xrt_run.get_ert_packet());// only for PS Kernels, will be removed after PS kernel migration to xrt::kernel   
```

Comment:
Cf other similar feedback

Reply:
seeing compilation error if we use static_cast here.  using auto now, not repeating the type. submitted latest changes in the PR.